content/release-10-14-0/announcements-10-14-0.md

```diff
@@ -50,9 +50,9 @@ To improve the security of Cumulocity IoT, in a future release 10.15+ the admini
 
 ##### Improved security for OAI-Secure users
 
-With the 10.16+ release, when an administrator changes permissions inside a role, users who have this specific role assigned and use OAI-Secure will be logged out.
+With the 10.16+ release, when an administrator changes permissions inside a role, the change impacts users who have the specific role assigned, and use OAI-Secure.
 
-The purpose of this measure is to enhance security and to make sure that all OAI-Secure users have access to the relevant information in Cumulocity IoT, without compromising secure access to the information. We recommend that administrators announce this change and notify about the change, because the logged-in users will immediately be logged out, and all their changes will get lost. Therefore this action requires scheduling.
+The purpose of this measure is to enhance security and to make sure that all OAI-Secure users have access to the relevant information in Cumulocity IoT, without compromising secure access to the information. We recommend that administrators announce this change and notify about the change, because the logged-in users will be affected and if they try to perform an operation that is affected by the changed role, they will receive access denied, and will not be able to perform the operation. Therefore this action is recommended to be followed up with notification to the users, and scheduling.
```

Comment:
```suggestion
The purpose of this measure is to enhance security and to make sure that all OAI-Secure users have access to the relevant information in Cumulocity IoT, without compromising secure access to the information. 

We recommend that administrators announce this change, because logged-in users will be affected. If they try to perform an operation that is affected by the changed role, they will receive an "access denied" error and will not be able to perform the operation. Therefore this action is recommended to be followed up with notification to the users and scheduling.
```